src/main/java/chatbot/util/Storage.java

```diff
@@ -0,0 +1,63 @@
+package chatbot.util;
+
+import java.io.BufferedInputStream;
+import java.io.BufferedOutputStream;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+
+/**
+ * Utility class to read and write to a file.
+ */
+public class Storage {
+    /**
+     * Saves a serializable object to the specified file.
+     *
+     * @param filePath the file to save to
+     * @param item     the serializable item to save
+     * @param <T>      the type of the serializable item
+     * @return returns true if the item was successfully saved, else returns false
+     */
+    public static <T extends Serializable> boolean Save(String filePath, T item) {
```

Comment:
Minor point, but I think it would better to name these two functions as `save` and `load` (camelCase), unless you have a reason for capitalizing them 😆

Reply:
Thanks for pointing that out. I just simply missed them out.